taxonomy-editor-frontend/src/pages/project/export/index.tsx

```diff
@@ -67,7 +67,8 @@ const ExportTaxonomyToGithub = ({
         sx={{ mt: 10, flexGrow: 1, textAlign: "center" }}
         variant="h5"
       >
-        Click the button below to export to GitHub
+        Click the button below to export to GitHub. Your changes will be
+        reviewed by the community before being accepted.
```

Comment:
thought: maybe these two sentences could be on different lines? and maybe the second one between parentheses?